developer_setup.md

```diff
@@ -199,6 +203,13 @@
 * Required Ruby version is 2.2.2+
 * Required Bundler version is 1.8.7+
 
+* Steps to install Ruby & Bundler using `rvm`
```

Comment:
Please remove this section, https://github.com/ManageIQ/guides/pull/253 fixes the extra `rbenv` documentation.